module-infrastructure/persistence-db/src/main/kotlin/site/yourevents/invitationinformation/repository/InvitationInformationRepository.kt

```diff
@@ -0,0 +1,16 @@
+package site.yourevents.invitationinformation.repository
+
+import org.springframework.stereotype.Repository
+import site.yourevents.invitationinformation.domain.InvitationInformation
+import site.yourevents.invitationinformation.entity.InvitationInformationEntity
+import site.yourevents.invitationinformation.port.out.InvitationInformationPersistencePort
+
+@Repository
+class InvitationInformationRepository(
+    private val invitationInformationJPARepository: InvitationInformationJPARepository
+) : InvitationInformationPersistencePort {
+    override fun saveInvitationInformation(invitationInformation: InvitationInformation): InvitationInformation {
+        return invitationInformationJPARepository.save(InvitationInformationEntity.from(invitationInformation))
+            .toDomain()
```

Comment:
적절한 개행을 통해 가독성을 높일 수 있을 것 같아요..!! 어떻게 생각하시나요?
```suggestion
        return invitationInformationJPARepository.save(
        InvitationInformationEntity.from(invitationInformation))
            .toDomain()
```

Reply:
앗!! 훨씬 좋아보입니다!! 개행 진행하였습니다!!